pkg/sfu/buffer/nack.go

```diff
@@ -1,33 +1,35 @@
 package buffer
 
 import (
+	"math"
 	"time"
 
 	"github.com/pion/rtcp"
 )
 
-const maxNackTimes = 3   // Max number of times a packet will be NACKed
-const maxNackCache = 100 // Max NACK sn the sfu will keep reference
-
-type nack struct {
-	seqNum       uint16
-	nacked       uint8
-	lastNackTime time.Time
-}
+const (
+	maxTries      = 5                      // Max number of times a packet will be NACKed
+	cacheSize     = 100                    // Max NACK sn the sfu will keep reference
+	maxWait       = 2 * time.Second        // how long to process a NACK before it is declared unrecoverable and purged
```

Comment:
Can we only use maxTries for this?

Reply:
Will do. Will get rid of this and also include the initial delay in the minInterval and simplify.